client/src/Components/Chart.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip} from 'recharts';
+
+const Chart = ({players}) => {
+    const data = players;
+
+    return (
+        <LineChart width = {1920} height = {300} data = {data} margin = {{top: 5, right: 30, left: 20, bottom: 5}}>
```

Comment:
Did meeting the stretch goal of changing the css of the charts help with this one?